pkg/installer/build.yml

```diff
@@ -0,0 +1,9 @@
+image: installer
+org: zededa
+network: no
+config:
+  binds:
+    - /dev:/dev
+  capabilities:
+    - all
+  pid: host
```

Comment:
Missing newline and end of file. Don't know if it matters

Reply:
vi people, they use line-based editors and pretend that the world is line based. EMACS FTW! (okay... I can change it).